### PR TITLE
fix: display better hotel images for test

### PR DESCRIPTION
**Pull request by valera-rozuvan**
If ENV variable `REACT_APP_USE_RND_HOTEL_IMG` is set to `true`, display random predefined hotel images instead of the black and white

```
"TEST IMAGE: This image is not a bug"
```

image.

### CODE REVIEW

**Comment by valera-rozuvan** [comment]
I am only not showing images which have the following sub-string in their URL:

```
'uat.multimediarepository.testing.amadeus.com'
'0796D584C500433995B3A5B5EF9F471E'
```

All other images - will show up on UI.

**Review by dave-ok — DISMISSED** [review]
Awesome feature!

File: src/containers/checkout/CheckoutSummary.tsx [hunk 1]
```diff
@@ -85,7 +86,7 @@ export const CheckoutSummary = () => {
               component="img"
               height="200"
               src={accommodationImage?.url}
-              fallback="/images/hotel-fallback.webp"
+              fallback={FallbackImage}
```
Comment (bobetbat) [inline src/containers/checkout/CheckoutSummary.tsx:89]: images should arrive from public folder

**Review by bobetbat — CHANGES_REQUESTED** [review]

File: src/components/ImageCarousel.tsx [hunk 2]
```diff
@@ -56,7 +60,17 @@ export const ImageCarousel: React.FC<{ media: MediaItem[]; ratio: ImageRatio }>
       >
         <Slider ref={carouselRef} {...settings} lazyLoad={'anticipated'}>
           {media && media.length > 0 ? (
-            media.map((item, i) => <Image key={i} src={item.url} ratio={ratio} />)
+            media.map((item, i) => {
+              let imgUrl: string | undefined = '';
+
+              if (thisIsTestImage(item.url)) {
```
Comment (silksil) [inline src/components/ImageCarousel.tsx:66]: is it maybe an idea, that you put this logic with the if and else statements in a separate function.

Something like `getAccommodationImage`. In that way we don't have to duplicate code and it is easier for others to implement this functionality

File: src/containers/checkout/CheckoutSummary.tsx (hunk 1 above)
Comment (valera-rozuvan) [reply src/containers/checkout/CheckoutSummary.tsx]: @bobetbat - fixed. PR is updated.

File: src/components/ImageCarousel.tsx (hunk 2 above)
Comment (valera-rozuvan) [reply src/components/ImageCarousel.tsx]: @silksil - fixed. PR is updated.

**Review by bobetbat — APPROVED** [review]
LGTM

**Pull request merged**